Default Display setting added

ccarpiog opened the pull request:
This change adds a setting that allows users to select a specific display, which will be automatically shown when the app opens.

ccarpiog committed:
Default Display setting added

amitv87 commented on CMakeLists.txt:
```diff
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 3.13)
 project(PiP)
 
-add_compile_options(-fobjc-arc -Wno-deprecated-declarations -Wno-format)
+add_compile_options(-fobjc-arc -Wno-deprecated-declarations -Wno-format -mmacosx-version-min=14.0)
```
Does it not build successfully for you without explictily adding that flag?
If it does not then add `set(CMAKE_OSX_DEPLOYMENT_TARGET "12.0.0")` as I still use monterey and remove the `macosx-version-min` flag.

ccarpiog committed:
Add custom display names feature

- Added UI panel to configure custom names for displays
- Custom names appear in Default Display selector and right-click menu
- Names are persisted in user preferences

ccarpiog committed:
Fix preferences UI and Display Names panel issues

- Add visual spacing between Display Renderer, Default Display, and Display Names options
- Fix Shift+Tab keyboard navigation in Display Names panel by completing circular tab order
- Add visual cue for unknown displays (italic placeholder text)
- Fix display names not persisting when clicking OK by committing edits before close

ccarpiog committed:
Add camera audio capture and monitoring feature

- Capture audio from camera/microphone during video capture
- Add ring buffer for real-time audio playback via AudioUnit
- Add microphone permission handling (non-blocking)
- Add "Audio Monitoring" menu toggle when camera is active
- Audio enabled by default when camera has microphone

ccarpiog committed:
Fix display names persistence across reboots

- Use stable EDID-based identifiers (vendor-model-serial) instead of
  CGDirectDisplayID which changes between reboots
- Remove NSScreen fallback that could deadlock WindowServer
- Fix intValue to unsignedIntValue for proper display ID handling
- Maintain backwards compatibility with legacy preference format

ccarpiog committed:
Fix camera audio capture using AVCaptureAudioPreviewOutput

Replace manual AudioUnit-based audio handling with Apple's built-in
AVCaptureAudioPreviewOutput for camera audio monitoring. This fixes
garbled audio issues caused by format conversion problems.

Changes:
- Remove complex ring buffer and AudioUnit render callback
- Use AVCaptureAudioPreviewOutput for automatic format handling
- Simplify audio toggle to volume control (0.0/1.0)
- Remove unnecessary audio processing in capture delegate

ccarpiog committed:
Add Shift+Tab navigation in preferences text fields

ccarpiog committed:
Close app when window is closed with red button

ccarpiog committed:
Add multi-window support (Phase 1 + Phase 2)

- Multiple PiP windows with independent sources via Cmd+N
- Shared MTLDevice across all windows to reduce VRAM usage
- Window management: grid layout (Cmd+G), cascade, close all (Opt+Cmd+W)
- Clone window (Shift+Cmd+N) with camera exclusivity warning
- Window manager panel (Opt+Cmd+M) showing source name, type, status
- Blank window overlay hint for source selection
- Disconnect/error handling for displays, cameras, and captured windows
- New preferences: max windows limit, new window behavior (blank/clone)
- Performance warning when 6+ windows open
- Fix NSPanel not counted by applicationShouldTerminateAfterLastWindowClosed
- Manual last-window termination check in windowWillClose
- Prevent panel auto-hide during permission dialogs (hidesOnDeactivate=NO)

ccarpiog committed:
Translate all UI text from Spanish to English and update LICENSE

ccarpiog committed:
Add HLS streaming pipeline and stabilize live delivery

ccarpiog committed:
Avoid shutdown deadlock in frame capture

ccarpiog committed:
Fix HLS stability and add outgoing camera audio

ccarpiog committed:
Translated streaming menu to English

ccarpiog committed:
Update README with live streaming documentation

ccarpiog committed:
Add unified default source with camera naming and sorting

ccarpiog committed:
Reduce HLS streaming latency to ~2-3 seconds

ccarpiog committed:
Separate local audio monitoring from HLS audio streaming

The audio monitoring toggle now only mutes/unmutes local playback
without affecting audio capture or HLS streaming to other devices.

ccarpiog committed:
Copy URL now points directly to the HLS stream

The Copy URL action and auto-copy on stream start now copy
the direct stream URL (with /stream.m3u8) instead of the
base viewer URL. Open in Browser still opens the web viewer.

ccarpiog committed:
Rename streaming menu items to clarify direction

Stream HLS -> Open Stream URL (incoming), Stream/Start Streaming ->
Broadcast/Broadcast This Window (outgoing)

ccarpiog committed:
Add Tailscale Funnel remote-access design doc

Plan and architecture for exposing the HLS stream outside the LAN via
Tailscale Funnel, with app-controlled enable/disable, per-user password
auth (one session per user, take-over on second device), and
duration-based user enablement. Ignore local agent instruction files
(CLAUDE.md, AGENTS.md).

ccarpiog committed:
Ignore local agent instruction files (CLAUDE.md, AGENTS.md)
